src/cli/telemetry/telemetry.ts

```diff
@@ -159,8 +160,10 @@ type ConfigStringKeys = keyof {
   [K in keyof d.Config as Required<d.Config>[K] extends string ? K : never]: d.Config[K];
 };
 
+const OUTPUT_TARGET_KEYS_TO_KEEP = ['type', 'typesDir', 'baseUrl'];
+
 /**
- * Anonymize the config for telemetry, replacing potentially revealing config props
+ * Anonymize the config for telemetry, replacing potentially revealing configprops
```

Comment:
Minor, was this 'configprops' change intentional in the most recent commit? I don't see any other usages of this spelling anywhere